### ISSUE TITLE
How to export character animation when using duik and joystick

### ISSUE THREAD

**Issue by Danjuanlab** [airbnb/lottie-web#1680]
Recently I tried to make complex animations using duik and joystick at same time. When I finished the bone binding I tried to export it, but I couldn't successfully build all the SVG animation elements in the demo browser. The browser prompts “Cannot read property 'length' of undefined.”

My export settings are below. Pdd s down in the animation is the version of IK. Backup is not done IK, but the bones and joystick are bound. Both will report the same error. I can't think of this problem myself. I hope hernan will help me to see where the problem.
![image](https://user-images.githubusercontent.com/42509669/59694972-0a3cb100-921c-11e9-8ab2-494fd996daad.png)
[pdd rig.zip](https://github.com/airbnb/lottie-web/files/3302094/pdd.rig.zip)

**Comment by bodymovin** [comment]
Hi, I'll try to check it this afternoon.
If you can provide a version as reduced as possible, it would help :)

**Comment by Danjuanlab** [comment]
I think I found where the problem is.
In this project, I used the joystick's switch template for the left hand, left leg and left ear. A common use is to copy the shape layer into the switch template. But in this case, I have to copy the pre-compose of the arm and foot into the switch template to achieve the goal. Because the arms and feet are designed separately, it contains the lower arms and joints of the upper arms.
So it should be that when bodymovin get into the switch template when building the element, the default is the shape layer, but at this time it is pre-composited. So It need to do another for function, So that throw the error.

**Comment by bodymovin** [comment]
yes, that's probably it, bodymovin doesn't support targeting nested comps in external comps.
If you have a simple version of the issue, I can try to support it in the near future.

**Comment by Danjuanlab** [comment]
> yes, that's probably it, bodymovin doesn't support targeting nested comps in external comps.
> If you have a simple version of the issue, I can try to support it in the near future.

[debug.zip](https://github.com/airbnb/lottie-web/files/3308371/debug.zip)
I made a lite version for you.
